Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/entity/proxy/AbstractEntityDocRetrieveProxy.java

```diff
@@ -45,8 +45,7 @@ protected String getUrl(String serviceName) {
         try {
             result = getWebServiceProxyHelper().getUrlLocalHomeCommunity(serviceName);
         } catch (Exception ex) {
-            LOG.warn("Unable to retreive url for service: " + serviceName);
-            LOG.warn("Error: " + ex.getMessage(), ex);
+            LOG.warn("Unable to retreive url for service: " + serviceName + " Error: " + ex.getMessage(), ex);
```

Comment:
Optional: Please use parameterize logging such as LOG.warm ("Unable to retrieve {} ",servcieName

Reply:
I'll let it go this time.